is_core/config.py

```diff
@@ -1,7 +1,20 @@
 from django.conf import settings as django_settings
 
 
+def _get_auth_login_view():
```

Comment:
Why is this being inferred like this and is not in the Django settings directly? Applies to other parts of the configuration as well.

Reply:
I think it's because of easier configuration, you don't need to do too much by hand.